### ISSUE TITLE
bug fixes, improvements roadmap for v0.5.0-c3

### ISSUE THREAD

**Issue by hoatle** [teracyhq/dev#428]
Should include:

- auto_create directory
- bug fix for override configuration

(optional)
- bug fix for internet connectivity?
- https improvement?
- caching?

**Comment by hoatle** [comment]
network problem, I'm still not sure about the root cause https://ubuntuforums.org/showthread.php?t=2301742

**Comment by phuonglm** [comment]
Does the dhclient eth0 fix that?

**Comment by hoatle** [comment]
I usually got lost internet connection after `$ vagrant reload --provision` and this cause a lot of problems. `$ dhclient eth0` got the internet back after `$ vagrant ssh` and this is annoying.

**Comment by hoatle** [comment]
I guess this could fix the dhclient problem:

```diff
diff --git a/Vagrantfile b/Vagrantfile
index 2f2dee3..ebf3f98 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -424,6 +424,14 @@ Vagrant.configure("2") do |config|
     "inline" => fix_hosts_command
   })
 
+  fix_dhclient_command = "sudo dhclient eth0"
+
+  provisioners.unshift({
+    "type" => "shell",
+    "name" => "fix-dhclient",
+    "inline" => fix_dhclient_command,
+    "run" => "always"
+  })
+
   # append ip shell as the last item to always display the ip address
   provisioners << {
     "type" => "shell",
```

**Comment by hoatle** [comment]
things are getting into the priority list:
- k8s setup support focus
- fix IP & hostmanager ip resolver bug
- bug fixes
- code clean up
- upgrade to the latest vagrant, virtualbox versions (optional)
- cluster support (optional)

**Comment by hoatle** [comment]
closed, we're moving to v0.6.0 instead because there are a lot of breaking changes for the upcoming releases.
